engine/script/src/script_module.cpp

```diff
@@ -231,6 +192,7 @@ namespace dmScript
         }
         free(value->m_Script);
         free(value->m_Name);
```

Comment:
Do we still need m_Name?

Reply:
Module.m_Name is if I understand correctly the name used in require("game.foo") and we use this for lookups each time a module is loaded. I'm not sure of the implications if we ditch m_Name in favor of m_Filename